includes/ziti/model_support.h

```diff
@@ -71,8 +71,8 @@ MODEL_API int cmp_##type(const type *lh, const type *rh); \
 MODEL_API void free_##type##_array(array(type) *ap);\
 MODEL_API int parse_##type(ptr(type) v, const char* json, size_t len);\
 MODEL_API int parse_##type##_ptr(ptr(type) *p, const char* json, size_t len);\
-MODEL_API int parse_##type##_array(array(type) *a, const char* json, size_t len);\
-MODEL_API int json_from_##type(const ptr(type) v, char *buf, size_t maxlen, size_t *len);
+MODEL_API int parse_##type##_array(array(type) *a, const char* json, size_t len); \
+MODEL_API char* type##_to_json(const ptr(type) v, int flags, size_t *len);
```

Comment:
I'm guessing some flags will be added in the future? What are you thinking about for this? I'll put in a vote for the ability to return "pretty" json or not - or even just json with no newlines added.

Reply:
yes, that's the intent